tests/zfs-tests/tests/functional/features/large_dnode/large_dnode_005_pos.ksh

```diff
@@ -42,17 +44,21 @@ function cleanup
 	fi
 
 	rm -f $TEST_STREAM
+	rm -f $TEST_STREAMINCR
 }
 
 log_onexit cleanup
 
 log_assert "zfs send stream with large dnodes accepted by new pool"
 
 log_must $ZFS create -o dnodesize=1k $TEST_SEND_FS
-log_must touch /$TEST_SEND_FS/$TEST_FILE
-log_must $ZFS umount $TEST_SEND_FS
```

Comment:
I think we need everything mounted until the last `diff -r` on line 71, just before we `cleanup()` (which actually `zfs destroy -r`)